code/modules/mob/living/living.dm

```diff
@@ -443,6 +443,12 @@
 	if (s_active && !(s_active.ClickAccessible(src, depth=STORAGE_VIEW_DEPTH) || s_active.Adjacent(src)))
 		s_active.close(src)
 
+	for(var/mob/M in oview(src))
```

Comment:
>istypeless

Would that include ghosts then? I just realized its been tracking ghosts this whole time, which seems a bit pointless